billingbudgets/.repo-metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "billingbudgets",
+  "name_pretty": "Cloud Billing Budget",
+  "product_documentation": "https://cloud.google.com/billing/docs/apis",
```

Comment:
The budget-oriented APIs we are wrapping here aren't documented below this URL.

Reply:
I think the actual docs may not be published. Checking with the tech writer to see where the docs will eventually be published

Reply:
Confirmed that the link will be https://cloud.google.com/billing/docs/how-to/budget-api-overview